repo2docker_wholetale/stata.py

```diff
@@ -65,8 +67,104 @@ def get_build_scripts(self):
                 sed -i "s/stata-mp/stata/g" /home/jovyan/.stata_kernel.conf
                 """,
             ),
+            (
+                "root",
+                r"""
+                sed -i "s/tray = yes/tray = no/g" /etc/xpra/conf.d/05_features.conf
+                """,
+            ),
+            (
+                "${NB_USER}",
+                r"""
+                mkdir -p ${HOME}/Desktop && \
+                printf "[Desktop Entry]\n\
+                Version=1.0\n\
+                Type=Application\n\
+                Name=STATA\n\
+                Comment=\n\
+                Exec=xstata\n\
+                Icon=/usr/local/stata/stata16.png\n\
+                Path=${HOME}/work/workspace\n\
+                Terminal=false\n\
+                StartupNotify=false\n"\
+                > ${HOME}/Desktop/STATA.desktop && \
+                printf "[Desktop Entry]\n\
+                Version=1.0\n\
+                Type=Application\n\
+                Name=Terminal\n\
+                Comment=\n\
+                Exec=exo-open --launch TerminalEmulator\n\
+                Icon=utilities-terminal\n\
+                Path=${HOME}/work/workspace\n\
+                Terminal=false\n\
+                StartupNotify=false\n"\
+                > ${HOME}/Desktop/Terminal.desktop && \
+                printf "[Desktop Entry]\n\
+                Version=1.0\n\
+                Type=Application\n\
+                Name=Firefox\n\
+                Comment=\n\
+                Exec=firefox %u\n\
+                Icon=firefox\n\
+                Path=\n\
+                Terminal=false\n\
+                StartupNotify=false\n"\
+                > ${HOME}/Desktop/Firefox.desktop && \
+                chmod +x ${HOME}/Desktop/*.desktop
+                """,
+            ),
+            (
+                "root",
+                r"""
+                mkdir -p /usr/local/stata/ado/plus && \
+                mkdir -p /usr/local/stata/ado/site && \
+                sleep 60 && \
```

Comment:
why sleep?

Reply:
Ack. Trash attempt to debug a build issue.